README.md

```diff
@@ -275,15 +275,15 @@ options:
 
  * xml-dangerous-qweb-replace-low-priority
 
-    - https://github.com/OCA/odoo-pre-commit-hooks/blob/v0.0.21/test_repo/broken_module/template1.xml#L18 Dangerous use of "replace" from view with priority `0 < 99`
-    - https://github.com/OCA/odoo-pre-commit-hooks/blob/v0.0.21/test_repo/broken_module/template1.xml#L4 Dangerous use of "replace" from view with priority `0 < 99`
-    - https://github.com/OCA/odoo-pre-commit-hooks/blob/v0.0.21/test_repo/broken_module/template1.xml#L7 Dangerous use of "replace" from view with priority `0 < 99`
-    - https://github.com/OCA/odoo-pre-commit-hooks/blob/v0.0.21/test_repo/broken_module/template1_copy.xml#L18 Dangerous use of "replace" from view with priority `0 < 99`
-    - https://github.com/OCA/odoo-pre-commit-hooks/blob/v0.0.21/test_repo/broken_module/template1_copy.xml#L4 Dangerous use of "replace" from view with priority `0 < 99`
-    - https://github.com/OCA/odoo-pre-commit-hooks/blob/v0.0.21/test_repo/broken_module/template1_copy.xml#L7 Dangerous use of "replace" from view with priority `0 < 99`
-    - https://github.com/OCA/odoo-pre-commit-hooks/blob/v0.0.21/test_repo/broken_module/template1_copy2.xml#L18 Dangerous use of "replace" from view with priority `0 < 99`
-    - https://github.com/OCA/odoo-pre-commit-hooks/blob/v0.0.21/test_repo/broken_module/template1_copy2.xml#L4 Dangerous use of "replace" from view with priority `0 < 99`
-    - https://github.com/OCA/odoo-pre-commit-hooks/blob/v0.0.21/test_repo/broken_module/template1_copy2.xml#L7 Dangerous use of "replace" from view with priority `0 < 99`
+    - https://github.com/OCA/odoo-pre-commit-hooks/blob/v0.0.21/test_repo/broken_module/template1.xml#L18 Dangerous use of "replace" from view with priority `0 < 99`. Only replace as a last resort. Try position="attributes", position="move" or t-if="False" (invisible) first
```

Comment:
What do you mean by (invisible)?

Take into account when you use `t-if="False"`, the element is not only invisible (like in views), it's not even rendered

Reply:
Well, technically something not rendered is invisible haha, but I agree. I guess 

> @antonag32
> 
> The `tox -e update-readme` is an env running only in py3.10 (same than CI) in order to avoid different output bc py version variant
> 
> I just ran locally and it is working as expected
> 
> Could you check if it is running only from py3.10 in your local, please?

I am using Python 3.10.9 :thinking: but if it works on your machine then it probably has to do with my system, I will try running it on a docker container.

Reply:
Right, I will correct it.

Reply:
To explain myself a little bit more, for Odoo it's a bit different. When you define invisible field in standard views (tree, form, ...), fields are actually invisible, but they're retrieved and computed. For instance, it's a common situation to define an invisible field just to be able to use in another field's domain or attrs.

You may even see the node when using Fields View Get from the developer tools menu.

When you use `t-if`, elements are not even rendered, which mean nodes are not included in the HTML. The equivalent of invisible in HTML would be `style="display: none;"`, although I don't like so much that approach because, why would you send elements that are not going to be used?

Reply:
@luisg123v thanks for the extra information, yes it makes sense. In fact it may be an interesting idea to lint for unused xml fields, invisible fields that are not used in any domain/attrs :thinking: